platform/ios/src/MGLMapView.h

```diff
@@ -45,6 +45,31 @@ typedef NS_ENUM(NSUInteger, MGLAnnotationVerticalAlignment) {
 };
 
 /**
+ The mode used to track the user location on the map. Used with
+ `MGLMapView.userTrackingMode`.
+ */
+typedef NS_ENUM(NSUInteger, MGLUserTrackingMode) {
+    /** The map does not follow the user location. */
+    MGLUserTrackingModeNone              = 0,
+    /** The map follows the user location. */
```

Comment:
Panning also disables `MGLUserTrackingModeFollow`, reverting to `MGLUserTrackingModeNone`.